.run/Celery.run.xml

```diff
@@ -20,7 +20,7 @@
     <option name="ADD_SOURCE_ROOTS" value="true" />
     <EXTENSION ID="PythonCoverageRunConfigurationExtension" runner="coverage.py" />
     <option name="SCRIPT_NAME" value="$PROJECT_DIR$/env/bin/celery" />
-    <option name="PARAMETERS" value="-A posthog worker -B --scheduler redbeat.RedBeatScheduler --without-heartbeat --without-mingle --loglevel=DEBUG" />
+    <option name="PARAMETERS" value="-A posthog worker -B --scheduler redbeat.RedBeatScheduler --without-heartbeat --without-mingle --loglevel=DEBUG --pool=solo" />
```

Comment:
do we need a similar change anywhere else?

bin/start?

vscode run?

Reply:
How do other people run it? 😅 

I can check...

Reply:
I adjusted vscode and bin/start (hope the latter is not used somewhere non-local)

Reply:
ooh... ah, I remember breaking prod by assuming the `bin` folder was local... so at least for one command it's not only local!

Reply:
let's only do vscode and pycharm now I think about it